### ISSUE TITLE
[pem] Support creating JWT-signing keys (.p8 files) in pem

### ISSUE THREAD

**Issue by vbresults** [fastlane/fastlane#9256]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

Re-opening #7848 -- @ohayon there are so many benefits to using the new P8 format over the old one, please consider implementing

**Comment by KrauseFx** [comment]
Hey @vbresults, thanks for bringing up this feature request again, this sounds very interesting. I haven't used the new system myself, so I don't know too much details about it. You don't have to renew the key every year, right? So it's a one-time task for an app, instead of a yearly one? Why do you want to automate it? Do you have a high number of apps, that would need to be setup (including push) automatically using _fastlane_? 

This will give us a better understanding of the uses of the feature 👍 Thanks

**Comment by vbresults** [comment]
@KrauseFx Yes to all, quite a high and growing number of apps, spread across different accounts, and not far away from hitting the 100 mark (:

**Comment by bencollier** [comment]
This would be useful for us too. We have some people putting apps together with fastlane / Jenkins that never need to touch the dev centre :)

**Comment by vbresults** [comment]
@ohayon Any progress on this?

**Comment by ide** [comment]
Can this issue be closed since https://github.com/fastlane/fastlane/pull/9560 has landed? (Thanks for the PR @snatchev!)

**Comment by bencollier** [comment]
@ide it's just hit spaceship, not PEM yet though?

**Comment by ide** [comment]
Ah ok, I didn't realize it wasn't wired up to pem yet.

**Comment by snatchev** [comment]
@ide @vbresults we had some thinking to do where this functionality belongs, and if `pem` was the right tool for it. We decided that, at least for now, it should go in `pem`. I am hoping to have it wired up soon.

In the meantime, you may be able to automate some work by using spaceship directly.

`$ bundle exec fastlane spaceship`

after you login...

```ruby
key = Spaceship::Portal::Key.create(name: 'My New Push Key', apns: true)
File.write('push-jwt.p8', key.download)
```

**NOTE:** keys can be downloaded only once from the developer portal. If you lose it, you must re-create the key. You can only have 2 keys provisioned at any given time.
